src/store/modules/return/mutation-types.ts

```diff
@@ -0,0 +1,4 @@
+export const SN_RETURN = 'return'
+export const RETURN_LIST_UPDATED = SN_RETURN + '/LIST_UPDATED'
+export const RETURN_CURRENT_UPDATED = SN_RETURN + '/CURRENT_PRODUCT_ADDED'
```

Comment:
I think the RETURN_CURRENT_UPDATED and RETURN_CURRENT_PRODUCT_ADDED values are swapped

Reply:
Fixed it sir